### PR TITLE
Replace women with female.

### PR DESCRIPTION
**Pull request by ismailsunni**
CC @Charlotte-Morgan

### CODE REVIEW

File: safe/definitionsv4/fields.py [hunk 1]
```diff
@@ -350,16 +350,16 @@
...
+    'key': 'female_count_field',
+    'name': tr('Female Count'),
+    'field_name': 'female',
     'type': qvariant_whole_numbers,
     'length': default_field_length,
     'precision': 0,
     'absolute': True,
     'description': tr(
-        'Attribute where the number of women of the feature is located.'),
+        'Attribute where the number of female of the feature is located.'),
```
Comment (Charlotte-Morgan) [inline safe/definitionsv4/fields.py]: females

File: safe/definitionsv4/fields.py [hunk 2]
```diff
@@ -494,7 +494,7 @@
     'length': default_field_length,
     'precision': default_field_precision,
     'absolute': False,
-    'description': tr('Attribute where the ratio of women is located.'),
+    'description': tr('Attribute where the ratio of female is located.'),
```
Comment (Charlotte-Morgan) [inline safe/definitionsv4/fields.py]: females

**Comment by Charlotte-Morgan** [comment]
hi @ismailsunni - LGTM with minor change to make the word plural you need to use "females" in a couple of places.
Its ok to say female count and use the singular.
Women was already a plural :)

small english lesson :+1:

**Comment by ismailsunni** [comment]
@Charlotte-Morgan thanks for the English lesson :D. I have fixed them also. I will merge this then...

**Pull request merged**